src/main/java/com/google/devtools/build/lib/rules/cpp/CppHelper.java

```diff
@@ -885,6 +886,26 @@ public static Artifact createDefFileActions(
     return defFile;
   }
 
+  /**
+   * Create action for generating trivial DEF file without any exports, should only be used when
+   * targeting Windows.
+   *
+   * @param dllName The DLL name to be written into the DEF file, it specifies which DLL is required
+   *     at runtime
+   * @return The DEF file artifact.
+   */
+  public static Artifact createTrivialDefFileAction(RuleContext ruleContext, String dllName) {
```

Comment:
Seems name of function is not very good, because def file created by this function is suitable only for `.dll`. Maybe createTrivial**Dll**DefFileAction is better?

Reply:
I think the fact we are creating a DEF file implies it's for a Windows DLL, there's no DEF file for shared library on other platforms, right?

Reply:
I mean that def can be used for linking `.exe`, not only for `.dll`. But def file created by `createTrivialDefFileAction` not suitable for `.exe` creation, because of `LIBRARY ...` string, which doesn't work for `.exe`.  So I propose emphasize that def file suitable for `.dll`, not `.exe`.

Reply:
I see, thanks for the notice. Done.